src/main/java/com/javagirls/MangoRoom/controller/ReservationController.java

```diff
@@ -29,5 +39,19 @@ public Reservation addReservation(@RequestBody ReservationDto reservation) {
         return ResponseEntity.ok(message);
     }
 
+    @DeleteMapping("/reservations/{id}")
+    public void deleteReservation(@RequestBody ReservationDto reservationDto) {
```

Comment:
myślę że to body jest kompletnie zbędne, id wystarczy -> ale uwaga, powinnaś mieć w argumencie metody @PathVariable int id w przeciwnym wypadku to id nic nie znaczy i nie jest wykorzystywane.